test/blocks/merch-card-collection/merch-card-collection.test.js

```diff
@@ -161,7 +161,23 @@ describe('Merch Cards', async () => {
 
   it('should override cards when asked to', async () => {
     const el = document.getElementById('multipleFilters');
-    el.dataset.overrides = '/override-photoshop,/override-express';
+    setConfig({
+      ...conf,
+      mep: {
+        custom: {
```

Comment:
suggestion: `deferred` or `lazy` instead of `custom`?

Reply:
To me it's less talkative than custom. custom means in that context that it will not be handled by perso, which implies it's lazy/deferred, but also that something else should take care of it :)